client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -97,6 +98,29 @@ const EmailProvidersStackedComparison = ( {
 		setDetailsExpanded( Object.fromEntries( expandedEntries ) );
 	};
 
+	const onIntervalLengthPathHandlerResolver = ( newIntervalLength: IntervalLength ) => {
+		const selectedSiteSlug = selectedSite?.slug ?? '';
+
+		if (
+			[ 'email-purchase', 'email-home-single-domain', 'email-home-selected-domain' ].some(
+				( context ) => context === comparisonContext
+			)
```

Comment:
I think we should be able to simplify this condition like this:

```suggestion
			[ 'email-purchase', 'email-home-single-domain', 'email-home-selected-domain' ].includes( comparisonContext )
```

Reply:
I always forget about includes! ❤️